maplibre/src/context.rs

```diff
@@ -25,7 +25,7 @@ impl ViewState {
             window_size.width(),
             window_size.height(),
             cgmath::Deg(110.0),
-            100.0,
+            1.0,
```

Comment:
Let's reevaluate what we should choose here. Maybe we should just decrease fovy, which will then increase the camera height.